crates/sparrow-instructions/src/evaluators/list/collect_string.rs

```diff
@@ -101,7 +101,7 @@ impl CollectStringEvaluator {
                 .add_value(self.max, entity_index, input.map(|s| s.to_owned()));
             let cur_list = self.token.state(entity_index);
 
-            list_builder.append_value(cur_list.clone());
+            list_builder.append_value(cur_list.iter().copied());
```

Comment:
You can't copy the strings, so you need `cloned` for that case.

Reply:
Oops, clearly didn't test this one.